### DISCUSSION TITLE
Question about use:enhance

### DISCUSSION THREAD

**Discussion by SenneVanderAuwera** [sveltejs/kit#10821] [Q&A] [answered]
I have a feeling that i don't quite understand an aspect about `use:enhance`. If i use `use:enhance` to validate the data before sending it to the server, could that not be bypassed by just disabling Javascript in the browser? Am I missing something?

**Comment by aloker** [comment, marked as answer]
Yes, you're right, client side validation can easily be disabled. In fact, without JavaScript enabled, use:enhance doesn't do anything at all. 
Client side validation is primarily meant as a convenience for the user. You **must** (re-)validate the request on the server.
